saleor/graphql/api.py

```diff
@@ -19,6 +20,8 @@ class Query(graphene.ObjectType):
         Category, filterset_class=DistinctFilterSet,
         level=graphene.Argument(graphene.Int))
     category = graphene.Field(Category, id=graphene.Argument(graphene.ID))
+    page = graphene.Field(Page, id=graphene.Argument(graphene.ID))
```

Comment:
I expect that the common case will be to get the pages by their slug. ID is fine as it is standard API, but we also need to add `slug` argument here.